src/components/@core/action-buttons/delete.tsx

```diff
@@ -6,22 +6,26 @@ import {
   AlertDialogHeader,
   AlertDialogOverlay,
   Button,
-  useDisclosure
-} from "@chakra-ui/react";
-import { useLocalRouter } from "@components/@core/local-link";
-import DeleteIcon from "@icons/delete";
-import notification, { NotificationType } from "@utils/notification";
-import useTranslation from "next-translate/useTranslation";
-import React from "react";
+  useDisclosure,
+} from '@chakra-ui/react';
+import { useLocalRouter } from '@components/@core/local-link';
+import DeleteIcon from '@icons/delete';
+import notification, { NotificationType } from '@utils/notification';
+import useTranslation from 'next-translate/useTranslation';
+import React from 'react';
 
-import SimpleActionButton from "./simple";
+import SimpleActionButton from './simple';
```

Comment:
please use double quotes instead of single, please run lint script before pushing code

Reply:
Correction added.